apps/wizard/app_pages/map_brackets.py

```diff
@@ -0,0 +1,923 @@
+    mb.upper_bracket_open = st.toggle("Upper bracket open", mb.upper_bracket_open, help=_message)
+    mb.run(
+        reload_data_values=False,
+        reload_openness=False,
+        reload_all_brackets=True,
+        reload_rank=True,
+        reload_optimal_brackets=True,
+    )
+    mb.percentile_min, mb.percentile_max = st.slider(
+        label="Percentile range covered by brackets",
+        min_value=0,
+        max_value=100,
+        value=(MIN_PERCENTILE, MAX_PERCENTILE),
+        help="Currently, this only affects linear brackets",
+    )
+    mb.run(
+        reload_data_values=False,
+        reload_openness=False,
+        reload_all_brackets=True,
+        reload_rank=True,
+        reload_optimal_brackets=True,
+    )
+    if mb.percentiles_ignored:
+        st.warning(
+            "Minimum and maximum percentiles are identical. Percentiles are ignored. Consider broadening the percentile range."
+        )
+
+    # Select bracket type.
+    bracket_type_labels = {
+        # Add optimal choice
+        f"Optimal ({mb.brackets_optimal['optimal']})": mb.brackets_optimal["optimal"],
+        # Rest of the options
+        **{
+            value: value
+            for value in list(BRACKET_LABELS["log"].values())
+            + list(BRACKET_LABELS["linear"].values())
+            + list(BRACKET_LABELS["custom"].values())
+        },
+    }
+    # Create a bracket type selector with a select box.
+    bracket_type = st.selectbox(
+        "Select linear or log-like",
+        options=bracket_type_labels,
+        # NOTE: By default, the first option will be selected, which is expected to be the optimal one.
+        index=0,
+        # horizontal=True,
+    )
+    mb.bracket_type = bracket_type_labels[bracket_type]  # type: ignore
+    if mb.bracket_type == BRACKET_LABELS["custom"]["custom"]:
+        # Create an input text box for custom brackets.
+        try:
+            brackets_manual = st.text_input(
+                label="Enter custom brackets, e.g. [0, 10, 20, 30, 40]", value=json.dumps(mb.brackets_selected)
+            )
+            brackets_manual = json.loads(brackets_manual)
+            brackets_manual = [float(bracket) for bracket in brackets_manual]
+            if (np.diff(brackets_manual) < 0).any():
+                st.warning("Custom brackets are not monotonically increasing.")
+        except json.JSONDecodeError:
+            st.error("Invalid format for input brackets.")
+            st.stop()
+        mb.update_custom_brackets(brackets_manual)
+    elif mb.bracket_type in list(BRACKET_LABELS["log"].values()):
+        # Create a slider for positive values.
+        # TODO: Maybe have a toggle to be able to manually select negative and positive values separately.
+        if len(mb.brackets_positive) <= 1:
+            st.error("No brackets possible.")
+            st.stop()
+        min_selected, max_selected = st.select_slider(  # type: ignore
+            "Select a range of values:",
```

Comment:
Can use the same structure of the title of the other slider: `Value range`

It is implicit that the user needs to select (no need to say it). No need to add the `:`